preprocessing/DataCleanser.py

```diff
@@ -1,4 +1,8 @@
 from nltk.corpus import stopwords
+from tqdm import tqdm
+from langdetect import detect
+from langdetect import DetectorFactory
```

Comment:
Since lines 3 and four are from the same library, they can be merged into one line

Reply:
This one lol